params.go

```diff
@@ -162,14 +162,19 @@ func NewIdempotencyKey() string {
 	return fmt.Sprintf("%v_%v", now, base64.URLEncoding.EncodeToString(buf)[:6])
 }
 
+// SetIdempotencyKey sets a value for the Idempotency-Key header.
+func (p *Params) SetIdempotencyKey(val string) {
+	p.IdempotencyKey = &val
+}
+
 // SetStripeAccount sets a value for the Stripe-Account header.
 func (p *Params) SetStripeAccount(val string) {
-	p.StripeAccount = val
+	p.StripeAccount = &val
```

Comment:
OOC why is this better than `String(val)`? I guess they are ultimately the same thing but since I've use `String(xxx)` everywhere I wanted to ask

Reply:
So the reason I didn't bother with `String()` is just that those helpers are basically a workaround for the fact that there's no way to take an address of a literal value. In cases like this where that's not a problem, it seems fine to me to just take the address directly. Thoughts?